.github/workflows/test.yml

```diff
@@ -11,24 +11,45 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  build-openassetio:
+    name: Build OpenAssetIO
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/openassetio/openassetio-build
+    steps:
+      - uses: actions/checkout@v3
+      - name: Build
+        uses: ./.github/build_openassetio
+
   test:
     name: Test-Resolver
     runs-on: ubuntu-latest
+    needs: build-openassetio
     container:
       image: aswf/ci-vfxall:2022-clang13.1
     steps:
       - uses: actions/checkout@v3
 
-      - name: Build and install (Docker)
-        run: >
-          cmake -S . -B build &&
-          cmake --build build &&
+      - name: Get OpenAssetIO
+        uses: actions/download-artifact@v3
+        with:
+          name: OpenAssetIO Build
+          path: ./openassetio-build
+
+      - name: Build and install Resolver
+        run: |
+          cmake -S . -DCMAKE_PREFIX_PATH="./openassetio-build" -B build
+          cmake --build build
           cmake --install build
 
       - run: |
           python -m pip install -r tests/requirements.txt
 
       - name: Test
-        run: >
+        run: |
+          export PXR_PLUGINPATH_NAME=$GITHUB_WORKSPACE/build/dist/resources/plugInfo.json
+          export LD_LIBRARY_PATH=/usr/local/lib:$GITHUB_WORKSPACE/openassetio-build/lib64
```

Comment:
[consider]/[plan] I suspect the fact we're using `root` to build is the reason `/usr/local/lib` isn't on the library search path. We could try to build/run using the `aswfuser` user that is available in the container.

Reply:
All things being equal, i'd rather leave this as it stands, rather than opening up this specific can of worms again. I'll write a comment explaining why this is here.